modules/imgcodecs/test/test_tiff.cpp

```diff
@@ -1109,7 +1108,7 @@ TEST(Imgcodecs_Tiff_Modes, write_multipage)
     vector<Mat> pages;
     for (size_t i = 0; i < page_count; i++)
     {
-        const Mat page = imread(root + page_files[i]);
+        const Mat page = imread(root + page_files[i], IMREAD_REDUCED_GRAYSCALE_8 + (int)i);
```

Comment:
I understood your idea to force output type to grayscale, but why REDUCED?

Reply:
I understood your idea to force output type to grayscale, but why REDUCED?

to use smaller files to write and speed up the test. 

( the code reads 2 gray scale image and 4 color image)